etrago/appl.py

```diff
@@ -736,6 +737,9 @@ def run_etrago(args, json_path):
     # skip snapshots
     etrago.skip_snapshots()
 
+    # Temporary drop DLR as it is currently not working with sclopf
+    etrago.network.lines_t.s_max_pu = pd.DataFrame(index=etrago.network.snapshots, columns=etrago.network.lines.index, data=1.)
```

Comment:
As it is implemented, we will be dropping DLR also for LOPF or market_grid.

Reply:
It is also needed for market_grid as that method also used the sclopf. But I can adjust it for the lopf